samples/transformation-graph/Training.ipynb

```diff
@@ -0,0 +1,193 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Transformation Graph - Auto Featuring\n",
+    "This is a component that uses a technique of exploring the possible solutions oriented by a graph to perform AutoFeaturing.\n",
+    "\n",
+    "This notebook shows:\n",
+    "- how to use the [SDK](https://platiagro.github.io/sdk/) to load datasets, save models and other artifacts.\n",
+    "- how to declare parameters and use them to build reusable components."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Declare parameters and model hyperparameters\n",
+    "Components may declare (and use) these default parameters:\n",
+    "- dataset\n",
+    "- target\n",
+    "\n",
+    "Use these parameters to load/save datasets, models, metrics, and figures with the help of [PlatIAgro SDK](https://platiagro.github.io/sdk/). <br />\n",
+    "You may also declare custom parameters to set when running an experiment.\n",
+    "\n",
+    "Select the hyperparameters and their respective values to be used when using the component:\n",
+    "- date\n",
+    "- group\n",
+    "- budget"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# parameters\n",
+    "dataset = \"eucalyptus-2.csv\" #@param {type:\"string\"}\n",
```

Comment:
@lucaslzl por favor, troque para dataset = "eucalyptus" para que esse exemplo use um dos datasets padrão da plataforma. Dessa forma, uma pessoa que for rodar este noebook em outra instalação consegue fazê-lo sem erros.